src/loggers.jl

```diff
@@ -460,6 +460,12 @@ let
                         end
                         throw(exc)
                     end
+
+                    function $level(logger::Logger, msg::AbstractString, exc::Exception)
+                        msg *= sprint(showerror, exc)
```

Comment:
would it be nice to be able to write
```julia
warn(getlogger(), "Woops", SomeError("Nope."))
```
(note I've "forgotten" to add a space (i.e. `"Woops "`)) 

and have it automatically render with a space as `Woops SomeError...`

Reply:
Sure

Reply:
```
julia> e = DomainError(-1.0, "sqrt will only return a complex result if called with a complex argument. Try sqrt(Complex(x)).")
DomainError(-1.0, "sqrt will only return a complex result if called with a complex argument. Try sqrt(Complex(x)).")

julia> warn(getlogger(), "Woops", e)
[warn | root]: Woops: DomainError with -1.0:
sqrt will only return a complex result if called with a complex argument. Try sqrt(Complex(x)).
```

Reply:
Hmmm, maybe a newline would be better.